docs/lang/articles/basic/type.md

```diff
@@ -14,9 +14,9 @@ e.g., `i32` and `f64`.
 
 The _category_ can be one of:
 
-- `i` for signed integers, e.g. 233, -666
-- `u` for unsigned integers, e.g. 233, 666
-- `f` for floating point numbers, e.g. 2.33, 1e-4
+- `i` for signed integers, e.g. 24, -32
+- `u` for unsigned integers, e.g. 128, 256
+- `f` for floating point numbers, e.g. 3.14, 1.0, 1e-4
 
 The _digital number_ can be one of:
```

Comment:
In the first paragraph of `## Primitive types` we call this `a number of _precision bits_`, while here we refer to this as `_digital number_`. I'm not sure whether this slight inconsistency will affect readers.

Reply:
I think it is fine. the_ digital number_ is the number of _precision bits_. It is also explained after the definition.